### DISCUSSION TITLE
调整成twoside模式后出现很多空白页

### DISCUSSION THREAD

**Discussion by gongwen1205** [whutug/whu-thesis#118] [Q&A]
写的时候用oneside单页模式写的，但要求是单页页眉和双页页眉不一样，调整成双页模式后很多空白页，麻烦好心人帮帮~感谢

**Comment by sikouhjw** [comment]
https://github.com/whutug/whu-thesis/issues/44

**Reply by gongwen1205** [reply]
不好意思我还是要说下区别，用oneside模式写的话，页眉每一页都是一样的。学院要求是单页页眉论文题目，双页如下。麻烦答主看看呢
![image](https://user-images.githubusercontent.com/62161547/163192803-67691d3d-3e06-4e88-9612-407cd4989263.png)

**Reply by tanukihee** [reply]
> 硕博是必须双页打印的。

**Reply by gongwen1205** [reply]
是的，但页眉问题不知怎么解决

**Reply by tanukihee** [reply]
> 写的时候用oneside单页模式写的

你不用单页模式写不就没问题了

**Reply by gongwen1205** [reply]
我没看懂#44的解释，换成双页模式会有空白页的问题怎么解决，转成pdf也有空白页，手动去除吗？

**Reply by tanukihee** [reply]
解释就是，这不是个问题，空白页是故意且必须留下来的

**Reply by gongwen1205** [reply]
需要pdf版提交

**Reply by gongwen1205** [reply]
我的问题是我们需要提交pdf版，但twoside模式下空白页还会出现在pdf中，不知道我说清楚了没？

**Reply by sikouhjw** [reply]
> 我的问题是我们需要提交pdf版，但twoside模式下空白页还会出现在pdf中，不知道我说清楚了没？

https://github.com/whutug/whu-thesis/issues/117

**Reply by tanukihee** [reply]
我认为向导师说明「这些空白页是故意且有必要留下来的」并不困难，再者它也不是完全的空白，还留有页眉与页码，至少不会被当成 pdf 格式错误。如果有必要，也许我会考虑在空白页上加上「此空白页出于排版目的而故意留下」的电子水印

实在不行的话，请参考 #117

**Reply by gongwen1205** [reply]
用#117解决了，感谢两位~

**Reply by gongwen1205** [reply]
> > 我的问题是我们需要提交pdf版，但twoside模式下空白页还会出现在pdf中，不知道我说清楚了没？
> 
> #117

不好意思还要再请教下，今天编译的时候加了#117的命令还是有空白页是因为什么呢？
![1649899770(1)](https://user-images.githubusercontent.com/62161547/163296242-20dc5f17-4063-4fde-bd56-00c35cd2046c.png)

**Reply by sikouhjw** [reply]
> 不好意思还要再请教下，今天编译的时候加了#117的命令还是有空白页是因为什么呢？

放到导言区。
